src/Util/Image.cc

```diff
@@ -29,3 +29,7 @@ bool IsComputedStokes(int stokes) {
 bool IsComputedStokes(const std::string& stokes) {
     return IsComputedStokes(StokesValues[StokesStringTypes[stokes]]);
 }
+
+bool AreEqual(float val1, float val2) {
+    return fabs(val1 - val2) < 1e-6;
+}
```

Comment:
We should also use `std::numeric_limits<float>::epsilon()` instead of a hard-coded error value. (I moved this comment; I had it in the wrong place -- sorry!)